html/browsers/browsing-the-web/navigating-across-documents/initial-empty-document/window-open-aboutblank-navigate-immediately.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Navigations on window.open(about:blank) immediately after opening</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/helpers.js"></script>
+<body></body>
+<script>
+"use strict";
+const postMessageToOpenerOnLoad = `
+    window.onload = () => {
+      window.opener.postMessage("loaded", "*")
+    }
+  `;
+const url1 = "resources/code-injector.html?1&pipe=sub(none)&code=" + encodeURIComponent(postMessageToOpenerOnLoad);
+const url2 = "resources/code-injector.html?2&pipe=sub(none)&code=" + encodeURIComponent(postMessageToOpenerOnLoad);
+
+promise_test(async t => {
+  // Open a window with URL set to about:blank but navigate away from it immediately below.
+  const openedWindow = windowOpenAboutBlank(t);
+
+  // Navigate away from the initial empty document through setting location.href.
+  openedWindow.location.href = url1;
+  await waitForMessage(t, "loaded");
+  assert_equals(openedWindow.history.length, 1,
+    "history.length must be 1 after normal navigation on initial empty document");
```

Comment:
I'm surprised Chrome is failing this?

Reply:
Yeah this is failing because of https://bugs.chromium.org/p/chromium/issues/detail?id=524208#c22, which is being fixed in crrev.com/c/2794069